solutions/src/main/java/ru/sberbank/school/task06/CollectionUtils.java

```diff
@@ -0,0 +1,184 @@
+package ru.sberbank.school.task06;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Iterator;
+
+public class CollectionUtils<T> {
+
+    /**
+     * Добавление всех элементов одной коллекции в другую.
+     *
+     * @param source      источник
+     * @param destination потребитель
+     */
+    public static <T> void addAll(List<? extends T> source, List<? super T> destination) {
+        destination.addAll(source);
+    }
+
+    /**
+     * Инициализация листа с реализаций java.uXtil.ArrayList.
+     *
+     * @return пустой ArrayList
+     */
+    public static <T> List<T> newArrayList() {
+        return new ArrayList<>();
+    }
+
+    /**
+     * Поиск индекса элемента в списке.
+     *
+     * @param source список, в котором искать элемент
+     * @param o      элемент
+     * @return индекс элемента
+     */
+    public static <T> int indexOf(List<? extends T> source, T o) {
+        return source.indexOf(o);
+    }
+
+    /**
+     * Обрезание списка заданным числом элементов. limit({1, 2 ,3}, 2) => {1, 2}
+     *
+     * @param source список
+     * @param size   необходимое число элементов
+     * @return сокращенный список
+     */
+    public static <T> List<T> limit(List<T> source, int size) {
+        if(source.size() == 0 || source.size() < size) {
```

Comment:
WhitespaceAround: &#39;if&#39; is not followed by whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1110477318>*